dump-scripts/python/zend_to_horizen.py

```diff
@@ -8,15 +8,34 @@
 
 """
 This script transforms the balances data dumped from zend in the format requested for Horizen. 
-Specifically:
- - the zend addresses are Base58 decoded and the prefix is removed 
- - the balances are converted from satoshis (or "zennies") to weis.
+Most accounts will be restored in ZendBackVault contract and they will need to be explicitly claimed by the owners to 
+transfer their balances to an Ethereum address.
+Others will be directly mapped to Ethereum addresses, provided off-chain by the accounts owners, and they will be restored 
```

Comment:
Let's be more explicit..:
Others will be directly mapped to Ethereum addresses, provided off-chain by the accounts owners 
-> For some external partners (CEX) not able to generate claim message signatures, a direct mapping to Ethereum addresses will be provided offchain

Reply:
Done